controllers/fenceagentsremediation_controller_test.go

```diff
@@ -56,8 +52,9 @@ const (
 )
 
 var (
-	faPodLabels = map[string]string{"app.kubernetes.io/name": "fence-agents-remediation-operator"}
-	log         = ctrl.Log.WithName("controllers-unit-test")
+	faPodLabels  = map[string]string{"app.kubernetes.io/name": "fence-agents-remediation-operator"}
+	log          = ctrl.Log.WithName("controllers-unit-test")
+	underTestFAR = &v1alpha1.FenceAgentsRemediation{}
```

Comment:
I would prefer to avoid making this a package wide var...

Reply:
fixed [here](https://github.com/medik8s/fence-agents-remediation/compare/fb7e571984f5c43af2d91c20602fbf20a1ecb934..b63a143313dbfa4e3b6b3c826e54bb09910f4f6f#diff-fac0cf56dda42683c1e24b46c87cfa4cca46c444145ee69de9e264db50f10289R52)